packages/swing-store-lmdb/lmdbSwingStore.js

```diff
@@ -39,6 +39,12 @@ function makeSwingStore(dirPath, forceReset = false) {
     }
   }
 
+  function diskUsage() {
+    const dataFilePath = `${dirPath}/data.mdb`;
```

Comment:
Does LMDB put everything into a single file? That's awesome, I thought there might be a journal file to track too.

Reply:
LMDB has two files per "database", but one is a lock file that's always 8K, so I didn't bother to count it.

Reply:
righto, good choice